src/name/ip_address.rs

```diff
@@ -180,21 +171,27 @@ impl<'a> IpAddressRef<'a> {
 
 #[cfg(feature = "std")]
 fn ipv6_to_uncompressed_string(octets: [u8; 16]) -> String {
-    octets
-        .chunks(2)
-        .map(|octet| format!("{:02x?}{:02x?}", octet[0], octet[1]))
-        .collect::<Vec<String>>()
-        .join(":")
+    format!(
```

Comment:
I suppose this is a fine solution. I might have gone with pre-allocating a `String::with_capacity()` and then using `.write_fmt()` to write into it in a loop.

Reply:
I have no strong opinion on this one.

Reply:
What what you do with the `Result` when using `write_fmt`? `ipv6_to_uncompressed_string` is meant to be used on an `impl From`, so it cannot fail.

Reply:
Just `unwrap()` it -- after all, `format!()` is actually doing the same thing and also doing something like unwrapping.

Reply:
Addressed in 1b2484189da6464a3bc184fe602f4eade4ca15e8